app/jobs/workoff_arbiter.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../../config/deploy/docker/lib/aws_sdk_helpers'
```

Comment:
This needs to move below the copyright.  We have scripts for updating it, but only if it starts at the top of the file and is followed by an empty line.